src/main/java/jss/bugtorch/features/squidFix/FixSquidSound.java

```diff
@@ -0,0 +1,151 @@
+package jss.bugtorch.features.squidFix;
+
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import jss.bugtorch.util.StaticUtils;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.passive.EntitySquid;
+import net.minecraft.potion.Potion;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.util.Vec3;
+import net.minecraft.world.World;
+import net.minecraft.world.WorldServer;
+import net.minecraftforge.event.entity.living.LivingDeathEvent;
+import net.minecraftforge.event.entity.living.LivingEvent;
+import net.minecraftforge.event.entity.living.LivingEvent.LivingUpdateEvent;
+import net.minecraftforge.event.entity.living.LivingHurtEvent;
+
+import java.util.List;
+import java.util.Random;
+
+
+
+
+public class FixSquidSound {
+
+
+    boolean isPass(LivingEvent event) {
+        return !event.entity.worldObj.isRemote && EntitySquid.class.equals(event.entity.getClass());

```

Comment:
event.entity instanceof EntitySquid

Reply:
Yep but i want add this only for vanilla squids